facdb_build/sql/dcp_colp.sql

```diff
@@ -46,301 +47,301 @@ SET hash =  md5(CAST((t.*)AS text)),
 				ELSE geo_bbl
 			END),
 	facname = (CASE
-				WHEN (name = ' ' OR name IS NULL) AND usedec ~* 'office' THEN 'Offices'
-				WHEN (name = ' ' OR name IS NULL) AND usedec ~* 'no use' THEN 'City Owned Property'
-				WHEN name <> ' ' AND name IS NOT NULL THEN initcap(name)
-				ELSE initcap(REPLACE(usedec, 'OTHER ', ''))
+				WHEN (parcelname = ' ' OR parcelname IS NULL) AND usetype ~* 'office' THEN 'Offices'
+				WHEN (parcelname = ' ' OR parcelname IS NULL) AND usetype ~* 'no use' THEN 'City Owned Property'
+				WHEN parcelname <> ' ' AND parcalname IS NOT NULL THEN initcap(parcelname)
+				ELSE initcap(REPLACE(usetype, 'OTHER ', ''))
 			END),
-	factype = initcap(REPLACE(usedec, 'OTHER ', '')),
+	factype = initcap(REPLACE(usetype, 'OTHER ', '')),
 	facsubgrp = (CASE
 
 			-- Admin of Gov
 
-			WHEN usedec LIKE '%AGREEMENT%'
+			WHEN usetype LIKE '%AGREEMENT%'
 
-				OR usedec LIKE '%DISPOSITION%'
+				OR usetype LIKE '%DISPOSITION%'
 
-				OR usedec LIKE '%COMMITMENT%'
+				OR usetype LIKE '%COMMITMENT%'
 
 				OR agency LIKE '%PRIVATE%'
 
 				THEN 'Properties Leased or Licensed to Non-public Entities'
```

Comment:
Flagging.  No records are returned.  Need to look into if the key words need to be changed.